EXPLAINER.md

```diff
@@ -0,0 +1,227 @@
+# Web Publications Explainer
+
+
+## Introduction
+
+What is a web publication? The spec [says](https://w3c.github.io/wpub/#intro):
+
+> A Web Publication is a discoverable and identifiable collection of resources. Information about the Web Publication is expressed in a machine-readable document called a manifest, which is what enables user agents to understand the bounds of the Web Publication and the connection between its resources.
+
+Two things make web publications different from the “ordinary” web we know and love. First, a web publication may consist of multiple resources that form a logical whole. Moby-Dick might consist of 136 HTML files in a specified order, but it’s still a single work. So searching should search all 136 chapters.
+
+More importantly, users have a set of expectations about how such content should be presented in order to make it easy to read and understand. Users need to personalize the presentation, using the font and font size that make it easiest for them to read. They want it to be easy to go to the next chapter without interrupting the reading experience by hunting for a link to click. They might need a high- or low-contrast version of the content. They want to read while offline.
+
+Thus the goal of web publications is to make these features ("affordances") available to readers. We hope that someday browsers will provide these features, but in the meantime we have to use scripting. That's OK, because we'll learn a lot from trying, and that will help the spec get better.
```

Comment:
We may be more explicit than just 'scripting': we can say that we may see polyfills, full blown applications, etc, coming up, and the goal of the work is to make these choices possible.

Maybe it is worth referring to an MVP in this respect?

Reply:
I've added a mention of polyfills. 

I'm a little wary of mentioning the concept of a MVP. Perhaps it's better to start by explaining the whole picture without prioritizing features.